contracts/network/BancorNetwork.sol

```diff
@@ -151,6 +157,8 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
 
     bool private _depositingEnabled = true;
 
+    uint32 private _rewardsPPM;
```

Comment:
_polRewardsPPM?

Reply:
changed to polRewardsPPM for all instances of rewardsPPM (state, getter, setter, event)